daemon/src/lib.rs

```diff
@@ -1,5 +1,8 @@
 pub mod actors;
```

Comment:
not entirely sure if we really need all the modules listed here, or if a more tree-like import works as well.